SPECS/ModemManager/ModemManager.spec

```diff
@@ -80,6 +81,9 @@ make  %{?_smp_mflags} check
 %{_libdir}/libmm-glib.la
 
 %changelog
+* Tue Mar 22 2022 Pawel Winogrodzki <pawelwi@microsoft.com> - 1.18.6-3
+- Adding missing systemd service file to the default package.
```

Comment:
Can we add an explicit dependency on systemd-devel? I'd hate for us to be in a situation where some dependency stops pulling in systemd-devel and we end up with a build break because the unit file isn't generated.

Reply:
Good idea!